docs/manage/_get_support.rst

```diff
@@ -52,6 +52,20 @@ Videos are not playing
 Make sure to check the :ref:`system requirements <sys_reqs>` to see if you can support video playback. Please report any issues on our `Community Forums <https://community.learningequality.org/>`_, stating the operating system and browser you are using.
 
 
+Antivirus
+---------
+
+Some overzealous antivirus programs on Windows platform may preventively impede Kolibri or some of its components (for example ``python.exe``) from running correctly. If that happens you need to add them to the antivirus exclusion list. Below steps refer to the program **Avast**, but should be similar in other antivirus applications.
```

Comment:
I'm a fan :heart:

Reply:
I can't actually remember the last time I had antivirus on any of my devices... Must be at least 15 years :scream: :sunglasses: